tests/misc/tparseopt.nim

```diff
@@ -1,4 +1,9 @@
 discard """
+  knownIssue: ''' 
```

Comment:
At present this will disable the entire test, so marking it as a known issue isn't going to achieve what you want. Instead we should just establish sensible behavior and have the test and module work that way.

Reply:
It's hard to fix parseopt